static/index.html

```diff
@@ -2,129 +2,121 @@
-				},
-			});
-			speedometer.maxValue = MAX_SPEED;
-			speedometer.setMinValue(0);
-			speedometer.animationSpeed = 16;
-			speedometer.setTextField(document.getElementById("speed"));
-			speedometer.set(0);
-		}
+</head>
+<body onclick="location.reload()"> <!-- Tap anywhere to reload in case something breaks -->
+    <svg width="800px" height="480px" id="svgId">
+        <style>
+            .valueText {
+                font-family: Helvetica;
+                text-anchor: middle;
+                font-size: 24px;
+            }
+            .valueSpan {
+                font-family: Helvetica;
+                text-anchor: middle;
+                font-size: 24px;
+            }
+            .labelSpan {
+                font-family: Helvetica;
+                text-anchor: middle;
+                font-size: 16px;
+            }
+            .minText {
+                font-family: Helvetica;
+                text-anchor: end;
+                font-size: 16px;
+            }
+            .maxText {
+                font-family: Helvetica;
+                text-anchor: start;
+                font-size: 16px;
+            }
+            .gaugePath {
+                stroke-linecap: butt;
+                transform: rotate(135deg);
+                transition: stroke-dashoffset 500ms ease-in-out;
+            }
+        </style>
+    </svg>
+	<script src="config.js"></script>
+    <script src="gauge.js"></script>
+    <script src="faults.js"></script>
+    <script>
+        let batteryCurrent = new Gauge(config.batteryCurrent);
+        let motorCurrent = new Gauge(config.motorCurrent);
+        let solarCurrent = new Gauge(config.solarCurrent);
+        let minCellVolt = new Gauge(config.minCellVolt);
+        let soc = new Gauge(config.soc);
+        let speed = new Gauge(config.speed);
+        let time = new Gauge(config.time);
+        let packVolt = new Gauge(config.packVolt);
+        let minPackTemp = new Gauge(config.minPackTemp);
+        let maxPackTemp = new Gauge(config.maxPackTemp);
+        let motorTemp = new Gauge(config.motorTemp);
+        let maxCellVolt = new Gauge(config.maxCellVolt);
 
-		function clockTick() {
-			let d = new Date();
-			let hour = ((d.getHours() % 12) || 12);
-			//hour = (hour < 10 ? "&nbsp;" : "") + hour; // Pad hour
-			let colon = d.getSeconds() % 2 ? ":" : " ";
-			let min = (d.getMinutes() < 10 ? "0" : "") + d.getMinutes()
-			let amOrPm = d.getHours() >= 12 ? "pm" : "am";
-			document.getElementById("clock").innerHTML = hour + colon + min + amOrPm;
-		}
-		
-		function readSensor(endpoint, callback) {
-			let xhttp = new XMLHttpRequest();
-			xhttp.onreadystatechange = function() {
-				if (this.readyState == 4 && this.status == 200) {
-					callback(this.response);
-				} else {
-					// Display error to driver?
-				}
-			};
-			xhttp.open("GET", endpoint + "?" + Date.now(), true); // Date.now() prevents caching
-			xhttp.send();
-		}
+		/*
+		let faults = new FaultBox({
```

Comment:
Why is the faults code commented out?

Reply:
This is commented out while we port the Fault code over to svg.